internal/host/host.go

```diff
@@ -1441,15 +1447,23 @@ func (m *Manager) GetHostByKubeKey(key types.NamespacedName) (*common.Host, erro
 }
 
 func (m *Manager) UnRegisterHost(ctx context.Context, h *models.Host) error {
-	if err := common.DeleteHostFromDB(m.db, h.ID.String(), h.InfraEnvID.String()); err != nil {
-		return err
-	}
 	if h.ClusterID != nil {
```

Comment:
```suggestion
	if h.ClusterID == nil {
	     return err
	 }
	
```

Reply:
I do return an error at the end.  do you think it better this way ?